README.md

```diff
@@ -33,3 +33,9 @@ npm run dev
 npm run build
 npm run start
 ```
+
+### Docker
+```bash
+sudo docker build -t txtdot/txtdot:1.2.0 .
+sudo docker run --network host -d txtdot/txtdot:1.2.0
```

Comment:
Just asking: what `--network host` is for?  
I mean, not what it does, but does it increase performace because there is no internal network, or something like that; or it is useless?